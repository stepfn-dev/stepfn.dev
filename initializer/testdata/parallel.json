{
  "StartAt": "Parallel",
  "States": {
    "Parallel": {
      "Type": "Parallel",
      "Branches": [
        {
          "StartAt": "CallLambda",
          "States": {
            "CallLambda": {
              "Type": "Task",
              "Resource": "arn:aws:states:::lambda:invoke",
              "Parameters": {
                "Payload.$": "$",
                "FunctionName": "unix"
              },
              "End": true
            }
          }
        },
        {
          "StartAt": "Branch2",
          "States": {
            "Branch2": {
              "Type": "Task",
              "Resource": "arn:aws:states:::lambda:invoke",
              "Parameters": {
                "Payload.$": "$",
                "FunctionName": "branch2func"
              },
              "End": true
            }
          }
        }
      ],
      "End": true
    }
  }
}
