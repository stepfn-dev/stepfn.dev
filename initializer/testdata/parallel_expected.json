{
  "StartAt": "Parallel",
  "States": {
    "Parallel": {
      "Type": "Parallel",
      "Branches": [
        {
          "StartAt": "CallLambda",
          "States": {
            "CallLambda": {
              "Type": "Task",
              "Resource": "arn:aws:states:::lambda:invoke",
              "Parameters": {
                "Payload.$": "$",
                "FunctionName": "funcarn",
                "ClientContext": "eyJjdXN0b20iOnsiSWQiOiJkeW5hbW9pZCIsIkhhbmRsZXIiOiJ1bml4In19"
              },
              "End": true
            }
          }
        },
        {
          "StartAt": "Branch2",
          "States": {
            "Branch2": {
              "Type": "Task",
              "Resource": "arn:aws:states:::lambda:invoke",
              "Parameters": {
                "Payload.$": "$",
                "FunctionName": "funcarn",
                "ClientContext": "eyJjdXN0b20iOnsiSWQiOiJkeW5hbW9pZCIsIkhhbmRsZXIiOiJicmFuY2gyZnVuYyJ9fQ=="
              },
              "End": true
            }
          }
        }
      ],
      "End": true
    }
  }
}
